Make 404 page more appealing.

cgiradkar opened the pull request:
The 404 page was having a relative link, updated code for resolving the image correctly.

relates to #124 

Signed-off-by: Chetan Giradkar <cgiradka@redhat.com>

TomSweeneyRedHat commented:
@Fodoj I'm not sure what's going on, but I don't think the changes you've made will fix the problem.  They're good additions so I'd keep them, but if I go to: https://github.com/containers/podman.io/blob/master/_posts/2020-04-05-new.md and do "View file", the metadata is shown in a table format.  However, if I view file for your *-new.md file in the GitHub review, it shows it like a comment in md. 

For some reason the md files aren't resolving appropriately and I'm not sure why.  I figured the dashes that you had at the top of the form were some different ascii char or you'd a rogue tab instead of a space or something?

The only suggestion I have is to copy the 2020-04-05-new.md file to 2020-04-15-new.md and then carefully cut/paste your verbiage from 2020-04-14-new.md into it trying to change the info in the metadata at the top as little as possible.  Ditto your other file.

Sorry that I don't have a more concrete resolution for you, I stared at everything for 30 minutes and didn't come up with a quick fix.

Fodoj commented:
@TomSweeneyRedHat I copied the other file and also kept just one paragraph of text for announcement. No idea if it helps somehow. I did not see any changes as a result of this.

TomSweeneyRedHat commented:
Somehow it's still messed up @Fodoj .  I'll try putting it together a PR to see if I can straighten it out.

cgiradkar committed:
Make 404 page more appealing.

The 404 page was just having a plain old vanilla text.
To revamp it, added a sad seal picture to signify the same.
NotFound component was Swizzled via docusaurus and edited accordingly.

Signed-off-by: Chetan Giradkar <cgiradka@redhat.com>

TomSweeneyRedHat commented:
LGTM
